network-api/networkapi/templates/fragments/buyersguide/category_dropdown.html

```diff
@@ -10,7 +10,7 @@
                tw-mr-auto
                tw-font-sans
                tw-font-bold
-               tw-text-sm
+               tw-text-[15px]
```

Comment:
Although `.pni-category-nav a` defines a 15px font size, I'm thinking if it wouldn't be better to change it to 16px. This way we could use `tw-text-base` here (which is 16px) and avoid the one-off style...

Reply:
Hey @jhonatan-lopes , good suggestion! I thought about it but then recalled there are a bunch of PNI existing styles that need to be updated to adapt to our design system. For now I'm thinking that we just make them consistent across the PNI pages and clean them up altogether as a later phase.

Reply:
Sounds good